kbase-extension/static/kbase/js/common/cellComponents/tabs/infoTab.js

```diff
@@ -52,7 +50,7 @@ define(['common/format', 'common/utils'], (format, utils) => {
             if (!appSpec && arg.currentApp) {
                 appSpec = model.getItem(`app.specs.${arg.currentApp}`); // for bulk cells
             }
-            const appRef = [appSpec.info.id, model.getItem('app').tag].filter(toBoolean).join('/');
+            const appRef = [appSpec.info.id, model.getItem('app').tag].filter((v) => !!v).join('/');
```

Comment:
This pattern was present in a few modules:
1. import `utils.toBoolean` as a separate variable.
2. pass that to a `.filter()` call as the filter function.

Most of these modules re-implemented it by copying and pasting the `toBoolean` function 😞 

The more modern one liner `v => !!v` does the same thing. I compared and contrasted with a gamut of inputs and saw no difference, so I just refactored everything else that way.

Reply:
😖